sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/UsingCookiesTest.cs

```diff
@@ -35,68 +35,58 @@ namespace SonarAnalyzer.UnitTest.Rules
-        }
+                additionalReferences: GetAdditionalReferencesForNet46());
 
         [TestMethod]
         [TestCategory("Rule")]
-        public void UsingCookies_CS_Disabled()
-        {
+        public void UsingCookies_CS_Disabled() =>
             Verifier.VerifyNoIssueReported(@"TestCases\UsingCookies_Net46.cs",
                 new CSharp.UsingCookies(),
-                additionalReferences: GetAdditionalReferences_Net46());
-        }
+                additionalReferences: GetAdditionalReferencesForNet46());
 
         [TestMethod]
         [TestCategory("Rule")]
-        public void UsingCookies_VB_Net46()
-        {
+        public void UsingCookies_VB_Net46() =>
             Verifier.VerifyAnalyzer(@"TestCases\UsingCookies_Net46.vb",
                 new VisualBasic.UsingCookies(AnalyzerConfiguration.AlwaysEnabled),
-                additionalReferences: GetAdditionalReferences_Net46());
-        }
+                additionalReferences: GetAdditionalReferencesForNet46());
 
         [TestMethod]
         [TestCategory("Rule")]
-        public void UsingCookies_VB_NetCore()
-        {
-            Verifier.VerifyAnalyzer(@"TestCases\UsingCookies_NetCore.vb",
-                new VisualBasic.UsingCookies(AnalyzerConfiguration.AlwaysEnabled),
-                additionalReferences: GetAdditionalReferences_NetCore(Constants.DotNetCore220Version));
+        public void UsingCookies_VB_Disabled() =>
+            Verifier.VerifyNoIssueReported(@"TestCases\UsingCookies_Net46.vb",
+                new VisualBasic.UsingCookies(),
+                additionalReferences: GetAdditionalReferencesForNet46());
 
-        }
+        private static IEnumerable<MetadataReference> GetAdditionalReferencesForNet46() =>
+            FrameworkMetadataReference.SystemWeb;
 
+#else
         [TestMethod]
         [TestCategory("Rule")]
-        public void UsingCookies_VB_Disabled()
-        {
-            Verifier.VerifyNoIssueReported(@"TestCases\UsingCookies_Net46.vb",
-                new VisualBasic.UsingCookies(),
-                additionalReferences: GetAdditionalReferences_Net46());
-        }
+        public void UsingCookies_CS_NetCore() =>
+            Verifier.VerifyAnalyzer(@"TestCases\UsingCookies_NetCore.cs",
+                new CSharp.UsingCookies(AnalyzerConfiguration.AlwaysEnabled),
+                additionalReferences: GetNetCoreAdditionalReferences(Constants.DotNetCore220Version));
+
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void UsingCookies_VB_NetCore() =>
+            Verifier.VerifyAnalyzer(@"TestCases\UsingCookies_NetCore.vb",
+                new VisualBasic.UsingCookies(AnalyzerConfiguration.AlwaysEnabled),
+                additionalReferences: GetNetCoreAdditionalReferences(Constants.DotNetCore220Version));
+
+        private static IEnumerable<MetadataReference> GetNetCoreAdditionalReferences(string packageVersion) =>
```

Comment:
nitpick: to keep the same name style it should be GetAdditionalReferencesForNetCore